public/content/exjobbs/openspace/2025_bock_interactivevideoeditor.mdx

```diff
@@ -0,0 +1,30 @@
+---
+name: "Interactive Astronomy Video Editor"
+description: "Create a user interface that can be used to create and author interactions and flight path to be used in movie production from OpenSpace"
+location: "Norrköping"
+period: "Spring 2025"
+number_of_students: "1-2"
+contact: [ "alebo68", "Joakim Kilby" ]
+
+finished: false
+skills: [ "OpenSpace", "Visualization", "Qt" ]
+---
+
+## About OpenSpace
+[OpenSpace](https://openspaceproject.com) is open-source software that aims to visualize the entire known universe in accurate three-dimensional space.  It is mature science visualization software that can be used by anyone to tell stories about the cosmos and our place in it.
+
+## The Project
+Over the last years it has gotten more popular to generate prerendered movies from interactive software and OpenSpace is no exception. Most recently the [VAST](https://visualiseringscenter.se/en/show/stort/) movie that is screened at the Visualization Center and dozens of other planetariums was created based off an interactive flight path in OpenSpace. Currently, a movie about the Apollo+Artemis missions is in production using the same techniques. For flat screen images, many shorter explanatory videos have been made for [YouTube](https://www.youtube.com/watch?v=E8rel2-kLJA&t=370s) or [TED](https://www.ted.com/talks/juna_kollmeier_the_most_detailed_map_of_galaxies_black_holes_and_stars_ever_made?subtitle=en).
+
+For each of these projects, creating the movie was a cumbersome process that requires manually maneuvering the camera and interacting with OpenSpace in real-time. Due to potential user errors, this process has to be retried many times until a good result is created as OpenSpace currently lacks the editing tools to fine-tune existing flight paths.
+
+This project's aim is to remedy this by creating a tool that is able to create a timeline of actions that can be edited in a user friendly way and then saved into an existing format so that OpenSpace can play them back either in real-time or use them to render out a video sequence to be used on a flat-screen or in a dome theater. Conceptually, one possible way to achieve this would be to create a layer editor similar to Adobe Premiere or other video editing tools, but instead of video or image layers, different settings of OpenSpace are controlled and manipulated.
+
+The decision about which software stack to use for this project is still open, but would either be a Qt-based application or a React-based webapplication depending on the applicants skills and preferences.
```

Comment:
```suggestion
The decision about which software stack to use for this project is still open, but it would either be a Qt-based application or a React-based web application depending on the applicants' skills and preferences.
```